language/tools/move-cli/tests/testsuite/republish/args.exp

```diff
@@ -38,27 +38,27 @@ Command `publish -v --no-republish`:
 Compiling Move modules...
 error: 
 
-   ┌── src/M.move:2:8 ───
+   ┌── build/mv_interfaces/00000000000000000000000000000042/M.move:2:8 ───
```

Comment:
Yeah I flipped some things so sources/targets are checked before libraries/deps, because it makes the logic for shadowing a lot easier. As a result it looks like the dep is a duplicate definition of the target and nto the other way around. 

I was able to fix it for the address mapping case, but this one is a bit trickier to change back